### PR TITLE
Changed color info

### PR DESCRIPTION
**Pull request by planetoftheweb**
The bg-gray color doesn't exist. I suggest you changed this to bg-secondary or bg-dark. Also, the references to text-white is odd because it's not listed as a valid text color. I changed it to text-light. Also changed text-gray-dark to text-dark

### CODE REVIEW

File: docs/4.0/utilities/colors.md [hunk 1]
```diff
@@ -11,19 +11,19 @@ toc: true
 <p class="text-{{ color.name }}">.text-{{ color.name }}</p>{% endfor %}
 {% endexample %}
 
-Contextual text classes also work well on anchors with the provided hover and focus states. **Note that the `.text-white` class has no link styling.**
```
Comment (XhmikosR) [inline docs/4.0/utilities/colors.md]: This is unrelated to this PR.

File: docs/4.0/utilities/colors.md [hunk 2]
```diff
@@ -11,19 +11,19 @@ toc: true
...
 {% for color in site.data.theme-colors %}
-<p><a href="#" class="text-{{ color.name }}{% if color.name == "light" %} bg-gray{% endif %}">{{ color.name | capitalize }} link</a></p>{% endfor %}
+<p><a href="#" class="text-{{ color.name }}{% if color.name == "light" %} bg-secondary{% endif %}">{{ color.name | capitalize }} link</a></p>{% endfor %}
 {% endexample %}
 
 Similar to the contextual text color classes, easily set the background of an element to any contextual class. Anchor components will darken on hover, just like the text classes. Background utilities **do not set `color`**, so in some cases you'll want to use `.text-*` utilities.
 
 {% example html %}
 {% for color in site.data.theme-colors %}
-<div class="p-3 mb-2 bg-{{ color.name }} {% if color.name == "light" %}text-gray-dark{% else %}text-white{% endif %}">.bg-{{ color.name }}</div>{% endfor %}
```
Comment (XhmikosR) [inline docs/4.0/utilities/colors.md]: `text-white` exists.

**Review by XhmikosR — CHANGES_REQUESTED** [review]

File: docs/4.0/utilities/colors.md (hunk 2 above)
Comment (planetoftheweb) [reply docs/4.0/utilities/colors.md]: Although it technically exists, it isn't listed in the color list, so I think it should either be listed or taken out.

Comment (XhmikosR) [reply docs/4.0/utilities/colors.md]: You have another PR for that anyway.

**Comment by mdo** [comment]
`.text-white` will be added to the docs in #23877, so closing out.
